torch/csrc/jit/script/init.cpp

```diff
@@ -209,34 +171,13 @@ struct VISIBILITY_HIDDEN PythonModuleValue : public PythonValue {
       Method& m,
       const std::string& field) override {
     py::object member = getattr(loc, field);
-    return toSugaredValue(member, m, loc);
+    // note: is_constant = true because we consider that global properties
+    // on modules like math.pi or torch.float to be constants
+    // eventhough it is possible, though rare, for someone to mutate them
```

Comment:
I am going to put this stuff in the hybrid frontend language reference.  It will have a section explaining how values from python are interpreted when used in script.